modules/aws/state_manager/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  state_management_key = "global/${var.environment}/s3/terraform.tfstate"
```

Comment:
@suggestion: rather than use var.environment and get the user to manually put in the aws region I think it would be better to use data.environment and specify 
`data "aws_region" "environment"` 
and use this throughout the code.

This is because they could be in eu-west-2 but specify var.environment = "eu-west-2" which would then make the labels not match the actual region.